tests/keras_tests/feature_networks_tests/feature_networks/activation_decomposition_test.py

```diff
@@ -31,8 +32,11 @@ def __init__(self, unit_test, activation_function: str):
         self.activation_function = activation_function
         super().__init__(unit_test)
 
+    def get_fw_hw_model(self):
+        return get_quantization_disabled_keras_hw_model("activation_decomp_test")
+
     def get_quantization_config(self):
-        return mct.QuantizationConfig(enable_weights_quantization=False, enable_activation_quantization=False)
+        return mct.QuantizationConfig()
```

Comment:
We can remove it, and the test will run with the default config that is returned from its superclass.